cve_bin_tool/checkers/bash.py

```diff
@@ -14,4 +14,3 @@ class BashChecker(Checker):
     FILENAME_PATTERNS = [r"bash"]
     VERSION_PATTERNS = [r"Bash version ([0-9]+\.[0-9]+\.[0-9]+)"]
     VENDOR_PACKAGE = [("gnu", "bash")]
-    MODULE_NAME = "bash"
```

Comment:
Hm, I was thinking that we'd retain module name for debugging, but you're probably right and we should be using vendor_package instead.